styles.css

```diff
@@ -71,8 +78,201 @@ a {
   font-size: 15px;
 }
 
+.connect-links {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+}
+
 .social-icons {
   height: 22px;
   width: 22px;
   padding-right: 15px;
 }
+
+#projects {
+  background-color: #f7f7f9;
+  padding-top: 100px;
+  padding-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+}
+
+.project {
+  background-color: #fff;
+  margin-top: 30px;
+  margin-right: 24px;
+  margin-left: 24px;
+  border-radius: 16px 16px 16px 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.project-image {
+  display: flex;
+  justify-content: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 15px;
+}
+
+.project-content {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.project-heading {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.content-text {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+  display: inline;
+}
+
+.canopy {
+  color: #344563;
+}
+
+.count-vector {
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+.description-text {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-right: 19px;
+  color: #344563;
+}
+
+.language-used {
+  background-color: #ebebff;
+  color: #6070ff;
+  padding: 4px 12px 4px 12px;
+  font-size: 12px;
+  border-radius: 8px 8px 8px 8px;
+}
+
+.project-link {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+}
```

Comment:
- [ ] It is good that you adding the hover effect of the button as indicated by previous reviewer. However, there are still some required changes.
- In the design, the hover background color is `#6070FF`
- In the design, the pressed effect (active) of the button has a background color of `#2230D2;`
Kindly incorporate these changes.